src/view/pages/statement/components/header/StatementHeader.tsx

```diff
@@ -82,6 +82,9 @@ const StatementHeader: FC<Props> = ({
 	const titleFontSize = calculateFontSize(title, 16, 25);
 	const isAdmin = statement?.creatorId === user?.uid;
 
+	// Checks if enableNavigationalElements is allowed
+	const enableNavigationalElements = statement?.statementSettings?.enableNavigationalElements;
```

Comment:
Did you talk to @talyaron about this?

Reply:
Sorry for the delayed response, I had some problems with the internet, i'll send him a message, thanks for reviewing

Reply:
He won't be available until Sunday

Reply:
Alright so I'll talk to him when he returns

Reply:
I am here...